nursery/send-sms-on-android.yml

```diff
@@ -0,0 +1,24 @@
+rule:
+  meta:
+    name: send SMS on Android
+    namespace: communication/sms
+    authors:
+      - "@mr-tz"
+    scope: function
+    # att&ck:
+    #   - Mobile::SMS Control [T1582]
```

Comment:
we currently only lint for Enterprise techniques, not https://attack.mitre.org/techniques/mobile/

Reply:
seems reasonable to add, assuming it's low effort and maintenance. we could also wait until someone wants to use this and/or we have more substantial coverage.